ax/core/tests/test_utils.py

```diff
@@ -185,3 +189,14 @@ def test_feasible_hypervolume(self) -> None:
             },
         )
         self.assertEqual(list(feas_hv), [0.0, 0.0, 1.0, 1.0])
+
+    def test_get_model_times(self) -> None:
+        exp = get_robust_branin_experiment(num_sobol_trials=2)
+        fit_times, gen_times = get_model_trace_of_times(exp)
+        total_fit_time, total_gen_time = get_model_times(exp)
+        fit_times_not_none = [none_throws(elt) for elt in fit_times]
+        gen_times_not_none = [none_throws(elt) for elt in gen_times]
+        self.assertTrue([elt > 0 for elt in fit_times_not_none])
+        self.assertTrue([elt > 0 for elt in gen_times_not_none])
```

Comment:
This only asserts that the list is non-empty (i.e. this passes even if every list element is `False`) - is this what you want? Or do you want to assert that `all(elt > 0 for elt in fit_times_not_none)` is `True`?

Reply:
Oops, yes, I wanted the latter!